vmdb/app/models/miq_request.rb

```diff
@@ -11,14 +11,17 @@ class MiqRequest < ActiveRecord::Base
 
   serialize   :options, Hash
 
+  default_value_for(:message)       { |r| "#{r.class::TASK_DESCRIPTION} - Request Created" }
```

Comment:
did you verify that the sub classes are getting the right message w/ description?